### PR TITLE
I added more details in all the testclasses

### PR DESCRIPTION
**Pull request by Tiffany910**
I remove the issues

### CODE REVIEW

File: tests/kakuro/TestBoardCell.java [hunk 1]
```diff
@@ -31,8 +37,9 @@ public void testInputCell(){
         assertEquals(cell.getFirstValue(), inputNumber);
         assertEquals(cell.getType(), BoardCell.CellType.EMPTY);
     }
-
-    @Test
+    
+    
+    |@Test
```
Comment (binarytrails) [inline tests/kakuro/TestBoardCell.java]:
```
test-compile:
    [mkdir] Created dir: /home/runner/work/kakuro/kakuro/build/test
    [javac] Compiling 9 source files to /home/runner/work/kakuro/kakuro/build/test
    [javac] /home/runner/work/kakuro/kakuro/tests/kakuro/TestBoardCell.java:42: error: illegal start of type
    [javac]     |@Test
    [javac]     ^
    [javac] /home/runner/work/kakuro/kakuro/tests/kakuro/TestBoardCell.java:42: error: illegal start of type
    [javac]     |@Test
    [javac]      ^
    [javac] /home/runner/work/kakuro/kakuro/tests/kakuro/TestBoardCell.java:42: error: ';' expected
    [javac]     |@Test
    [javac]           ^
    [javac] 3 errors
```

File: tests/kakuro/TestBoardUI.java [hunk 2]
```diff
@@ -11,7 +11,8 @@
 import org.junit.Test;
 
 public class TestBoardUI {
-    @Test
+	//The testBoardUIGeneration will test if the board will be created base of the rows and columns defined. If this works, then a bigger board can be produced.
+	@Test
```
Comment (binarytrails) [inline tests/kakuro/TestBoardUI.java:15]: why the extra space misaligning the method & the decorator?

File: tests/kakuro/TestDBConnection.java [hunk 3]
```diff
@@ -24,7 +27,8 @@ public void testConnect(){
         //Assert
         db.disconnect();
 	}
-     
+    
+  //testDisconnect method will check if there is no issues while disconnecting to the database
```
Comment (binarytrails) [inline tests/kakuro/TestDBConnection.java]: why -1 space  less misaligning the method and comment?

File: tests/kakuro/TestPlayerModel.java [hunk 4]
```diff
@@ -8,9 +8,9 @@
 import org.junit.Test;
 import kakuro.PlayerModel;
 
-//iteration 3
-
-public class TestPlayerModel {
+	//iteration 3
```
Comment (binarytrails) [inline tests/kakuro/TestPlayerModel.java]: maybe it's good to remove iteration 3 comment cause it's confusing since we did it and it's in second

**Review by binarytrails — CHANGES_REQUESTED** [review]

File: tests/kakuro/TestBoardCell.java (hunk 1 above)
Comment (binarytrails) [reply tests/kakuro/TestBoardCell.java]: its a typo of | before the ```@Test``` decorator

File: tests/kakuro/TestPlayerModel.java (hunk 4 above)
Comment (Tiffany910) [reply tests/kakuro/TestPlayerModel.java]: Change it

File: tests/kakuro/TestBoardCell.java (hunk 1 above)
Comment (Tiffany910) [reply tests/kakuro/TestBoardCell.java]: Done

File: tests/kakuro/TestBoardUI.java (hunk 2 above)
Comment (Tiffany910) [reply tests/kakuro/TestBoardUI.java:15]: In my codes, its aligned so I am not very sure where the error is. I will work on it

Comment (binarytrails) [reply tests/kakuro/TestBoardUI.java:15]: the error is that you mix pre-existing 4 space we use everywhere for indent and tabs

**Review by binarytrails — COMMENTED** [review]
@Tiffany910 tests passes but it says: "This branch has conflicts that must be resolved"

**Comment by binarytrails** [comment]
Recovery in https://github.com/binarytrails/kakuro/pull/60

**Pull request closed without merging**